app/javascript/components/wifi.jsx

```diff
@@ -38,7 +38,7 @@ export const Wifi = () => (
         }
 
         const { wifiName, wifiPassword } = data.primaryLocation;
-        if (!wifiName) {
+        if (!wifiName || !wifiPassword) {
```

Comment:
Nice! This is a good logical expression when both wifiName and wifiPassword should have a non-NULL value.